### DISCUSSION TITLE
New Tab pages should have some more functionality

### DISCUSSION THREAD

**Discussion by qubistt** [dailydotdev/daily#545] [Feature Requests]
Hey! 
Really love what you're doing. I just love seeing all my favourite developer news in one place.
Since daily.dev is mainly a browser extension, you can design it in such a way, that we can keep some bookmarks, maybe a search bar, the time, along with some widgets.

Thanks,
Adv

**Comment by nimrodkra** [comment]
Hi @Qubisted thanks for the feedback! Once in a while, we do our best to add features that make the new tab better match the browser's default one. I think the biggest thing we have now that should probably help you overcome some of the things you mentioned is our custom links feature: https://changelog.daily.dev/custom-shortcuts-219544

**Reply by qubistt** [reply]
This is perfect for me! Thanks a ton.

:) Qubist
